internal/apps/dop/providers/issue/core/query/issueexcel/sheets/sheet_issue/export_template.go

```diff
@@ -57,7 +57,7 @@ func GenerateSampleIssueSheetModels(data *vars.DataForFulfill) []vars.IssueSheet
 	requirementCommon := common
 	requirementCommon.ID = 1
 	requirementCommon.IssueType = pb.IssueTypeEnum_REQUIREMENT
-	requirementCommon.Content = "这个一个需求，包含一个任务"
+	requirementCommon.Content = data.I18n(fieldRequirementContent)
 	requirementCommon.State = "进行中"
```

Comment:
Field `State` is this also a problem?

Reply:
The field `State` has not translated in other function，so i delete the translation for `State` 
![Uploading image.png…]()

Reply:
<img width="241" alt="image" src="https://github.com/erda-project/erda/assets/86157286/95e73521-eff2-4ed9-93f7-e744f595af6f">